### ISSUE TITLE
Crashes on android when selecting image

### ISSUE THREAD

**Issue by superandrew213** [ivpusic/react-native-image-crop-picker#16]
App crashes when selecting image on Android. Image picker opens fine, but once image is selected and select is pressed, it crashes.

Using RN0.26.2.

**Comment by ivpusic** [comment]
what android version? are you selecing your local or remote image?

**Comment by superandrew213** [comment]
Android version is 6.0, remote image, and here it's where it crashes (just after OpenGLRenderer):

```
07-03 08:31:08.838  2662  2679 D OpenGLRenderer: endAllStagingAnimators on 0x7f9ea968d800 (ListView) with handle 0x7f9eb56a2880
07-03 08:31:18.264  2662  2662 W ViewRootImpl: Cancelling event due to no window focus: MotionEvent { action=ACTION_CANCEL, actionButton=0, id[0]=0, x[0]=378.33618, y[0]=799.33594, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=329113, downTime=318875, deviceId=0, source=0x1002 }
07-03 08:31:18.264  2662  2662 W ViewRootImpl: Cancelling event due to no window focus: MotionEvent { action=ACTION_CANCEL, actionButton=0, id[0]=0, x[0]=378.33618, y[0]=799.33594, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=329113, downTime=318875, deviceId=0, source=0x1002 }
07-03 08:31:18.264  2662  2662 W ViewRootImpl: Cancelling event due to no window focus: MotionEvent { action=ACTION_CANCEL, actionButton=0, id[0]=0, x[0]=378.33618, y[0]=799.33594, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=329113, downTime=318875, deviceId=0, source=0x1002 }
07-03 08:31:18.264  2662  2662 W ViewRootImpl: Cancelling event due to no window focus: MotionEvent { action=ACTION_CANCEL, actionButton=0, id[0]=0, x[0]=378.33618, y[0]=799.33594, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=329113, downTime=318875, deviceId=0, source=0x1002 }
07-03 08:31:18.712  2662  2662 D Documents: onActivityResult() code=-1
07-03 08:31:18.754  2933  2933 D AndroidRuntime: Shutting down VM
07-03 08:31:18.754  2933  2933 E AndroidRuntime: FATAL EXCEPTION: main
07-03 08:31:18.754  2933  2933 E AndroidRuntime: Process: com.sensio.testing, PID: 2933
07-03 08:31:18.754  2933  2933 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=-1, data=Intent { dat=content://media/external/images/media/11 }} to activity {com.sensio.testing/com.sensio.testing.MainActivity}: java.lang.IllegalArgumentException: Invalid URI: content://media/external/images/media/11
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:3699)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:3742)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.ActivityThread.-wrap16(ActivityThread.java)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1393)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5417)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: Invalid URI: content://media/external/images/media/11
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.provider.DocumentsContract.getDocumentId(DocumentsContract.java:780)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.reactnative.picker.RealPathUtil.getRealPathFromURI_API19(RealPathUtil.java:27)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.reactnative.picker.RealPathUtil.getRealPathFromURI(RealPathUtil.java:21)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.reactnative.picker.PickerModule.getImage(PickerModule.java:108)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.reactnative.picker.PickerModule.onActivityResult(PickerModule.java:143)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.facebook.react.bridge.ReactContext.onActivityResult(ReactContext.java:198)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManagerImpl.onActivityResult(ReactInstanceManagerImpl.java:599)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at com.facebook.react.ReactActivity.onActivityResult(ReactActivity.java:180)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:6428)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:3695)
07-03 08:31:18.754  2933  2933 E AndroidRuntime: 	... 9 more
07-03 08:31:18.755  1522  1534 W ActivityManager:   Force finishing activity com.sensio.testing/.MainActivity
07-03 08:31:18.791  1522  2759 I OpenGLRenderer: Initialized EGL, version 1.4
07-03 08:31:19.257  1522  1536 W ActivityManager: Activity pause timeout for ActivityRecord{f6b312f u0 com.sensio.testing/.MainActivity t7 f}
07-03 08:31:19.496  2662  2662 I art     : Starting a blocking GC Explicit
07-03 08:31:19.506  2662  2662 I art     : Explicit concurrent mark sweep GC freed 3499(296KB) AllocSpace objects, 1(24KB) LOS objects, 24% free, 1835KB/2MB, paused 366us total 9.846ms
07-03 08:31:19.510  2662  2662 I art     : Starting a blocking GC Explicit
07-03 08:31:19.515  2662  2662 I art     : Explicit concurrent mark sweep GC freed 742(57KB) AllocSpace objects, 4(216KB) LOS objects, 25% free, 1561KB/2MB, paused 371us total 4.777ms
07-03 08:31:19.515  2662  2662 I art     : Starting a blocking GC Explicit
07-03 08:31:19.528  2662  2662 I art     : Explicit concurrent mark sweep GC freed 5(160B) AllocSpace objects, 0(0B) LOS objects, 24% free, 1562KB/2MB, paused 336us total 12.790ms
07-03 08:31:19.720  1522  1536 I ActivityManager: Killing 2094:com.android.email/u0a27 (adj 15): empty #17
07-03 08:31:19.851  2395  2459 W OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
07-03 08:31:19.851  2395  2459 W OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
07-03 08:31:29.494  1522  1536 W ActivityManager: Activity destroy timeout for ActivityRecord{f6b312f u0 com.sensio.testing/.MainActivity t7 f}
```

**Comment by superandrew213** [comment]
Getting different types of url

```
content://media/external/images/media/3951

content://com.android.providers.media.documents/document/image:3951
```

Please check [#17](https://github.com/ivpusic/react-native-image-crop-picker/pull/17) for fix. Let me know if it works fine on your end.

**Comment by superandrew213** [comment]
@ivpusic what do you think about the PR? The different uri happen on kitkat
http://stackoverflow.com/questions/19834842/android-gallery-on-kitkat-returns-different-uri-for-intent-action-get-content

**Comment by ivpusic** [comment]
thank you for reporting and fixing this. will check it today.

**Comment by ivpusic** [comment]
PR merged. Tnx a lot

**Comment by ivpusic** [comment]
available in version 0.4.2
